content/fr/_index.md

```diff
@@ -97,9 +97,9 @@ title: "InnerSource Commons"
         <a class="twitter-timeline" data-height="500" data-dnt="true" href="https://twitter.com/InnerSourceOrg?ref_src=twsrc%5Etfw"></a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
       </div>
       <div class="col-md-5 offset-md-1">
-        <p class="h2 section-title">Stay Connected</p>
-        <p class="mb-4">Most of the action in the community happens on our <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">Slack Channel</a>. You can also keep informed about the InnerSource Commons news, events and activities by signing up to our mailing list below, follow us on <a href="https://twitter.com/InnerSourceOrg">Twitter</a> and <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, and subscribing to our <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">YouTube Channel</a>.</p>
-        <p class="h3 section-title">Sign up to our mailing list</p>
+        <p class="h2 section-title">Rester Connecté</p>
+        <p class="mb-4">La plupart des activités de la communauté se déroulent sur le <a href="https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw">Canal Slack </a>. Vous pouvez également vous tenir au courant des nouvelles, des événements et des activités d'InnerSource Commons en vous inscrivant à notre liste de diffusion ci-dessous, suivez-nous sur <a href="https://twitter.com/InnerSourceOrg">Twitter</a> et <a href="https://www.linkedin.com/company/innersourcecommons">LinkedIn</a>, et abonnez vous sur notre <a href="https://www.youtube.com/channel/UCoSPSd6Or4F_vpjo4SmyoEA">Chaine YouTube</a>.</p>
```

Comment:
Chaîne YouTube

Reply:
Could be spelled Chaine or Chaîne (according to new reform, since 1990, ref https://www.dictionnaire-academie.fr/article/A9C1485), I prefer to avoid the version with ^ since I would have to change in multiple places.